be/src/olap/rowset/segment_v2/inverted_index_reader.h

```diff
@@ -407,6 +407,8 @@ class InvertedIndexQueryParamFactory {
             M(PrimitiveType::TYPE_CHAR)
             M(PrimitiveType::TYPE_VARCHAR)
             M(PrimitiveType::TYPE_STRING)
+            M(PrimitiveType::TYPE_DATEV2)
+            M(PrimitiveType::TYPE_DATETIMEV2)
```

Comment:
add testcase for TYPE_DATEV2 and TYPE_DATETIMEV2

Reply:
already have